src/api/db/job.ts

```diff
@@ -26,21 +14,24 @@ export function getJob(jobId: number): Promise<any> {
       .then(job => {
         getBuild(job.builds_id)
```

Comment:
why you are making additional two queries in database if you get all data with the initial `withRelated: ['build.repository', 'runs']`?